spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/reactor/ReactorSleuth.java

```diff
@@ -154,7 +163,7 @@ private static boolean contextRefreshed(BeanFactory beanFactory) {
 				beanFactory,
 				sub,
 				sub.currentContext(),
-				scannable.name()) {
+				null) {
```

Comment:
it is used to name the spans. I suggest to pass `scannable` and call `scannable.name()` only when necessary